src/sentry/api/endpoints/organization_metrics_query.py

```diff
@@ -166,13 +168,31 @@ def post(self, request: Request, organization: Organization) -> Response:
                 },
             )
 
+            if all(
+                features.has("projects:use-eap-spans-for-metrics-explorer", project)
+                for project in projects
+            ):
+                if len(mql_queries) == 1 and "a" in mql_queries[0].sub_queries:
+                    subquery = mql_queries[0].sub_queries["a"]
+                    if "d:eap/" in subquery.mql:
+                        results = mql_eap_bridge.make_eap_request(
+                            subquery.mql,
+                            start,
+                            end,
+                            interval,
+                            organization,
+                            projects,
+                            Referrer.API_ORGANIZATION_METRICS_QUERY.value,
```

Comment:
I would make this a separate referrer, to make it a little easier to track independently in Snuba.

Reply:
done